ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -273,8 +273,9 @@ ScrollView {
                         showMutedContacts: true
                     })
                     //% "Muted contacts"
-                    mutedChatsModal.title = qsTrId("muted-contacts")
-                    mutedChatsModal.open()
+                    mutedChatsModal.title = qsTrId("muted-contacts");
+                    mutedChatsModal.noContentText = qsTrId("Muted contacts will appear here");
```

Comment:
You need to use `qsTr` when using normal strings. We have a translation script that will modify `qsTr` to `qsTrId`
```suggestion
                    mutedChatsModal.noContentText = qsTr("Muted contacts will appear here");
```

Reply:
Right, updated!

Reply:
Yeah that sounds like a bug that needs to be fixed